algorithms.tex

```diff
@@ -1560,10 +1563,10 @@ \section*{25.?? Tag specifiers\hfill[alg.tagspec]}\label{alg.tagspec}
 \pnum
 \requires \tcode{Function} shall meet the requirements of
 \tcode{MoveConstructible}~(\changed{Table~\cxxref{moveconstructible}}{
-\ref{concepts.lib.corelang.moveconstructible}}).
+\ref{concepts.lib.object.moveconstructible}}).
 \enternote \tcode{Function} need not meet the requirements of
 \tcode{CopyConstructible}~(\changed{Table~\cxxref{copyconstructible}}{
-\ref{concepts.lib.corelang.copyconstructible}}).\exitnote
+\ref{concepts.lib.object.copyconstructible}}).\exitnote
```

Comment:
We decided the 'structables are feature concepts. Do we still want to list them in a section for "object" concepts?